apps/modernization-ui/src/components/Link/Link.tsx

```diff
@@ -0,0 +1,13 @@
+import './Link.scss';
+
+type LinkProps = { name: string; link: string; className?: string };
+
+const Link = ({ name, link, className = '' }: LinkProps) => {
```

Comment:
issue:  Might have gone too generic.  This should be an encapsulation of menu items of the Horizontal Navigation menu.  This should include the `div` and any styles needed.